src/runtime_src/tools/scripts/pkgapu.sh

```diff
@@ -27,10 +27,12 @@ usage()
 	echo "  options:"
 	echo "          -help                           Print this usage"
 	echo "          -images                         Versal images path"
-	echo "          -petalinux                      Petalinux path"
 	echo "          -clean                          Remove build files"
         echo "          -output                         output path"
-	echo ""
+	echo "This script requires tools: mkimage, xclbinutil, bootgen, rpmbuild, dpkg-deb. "
```

Comment:
You might wish to check to see if the XILINX_XRT and XILINX_VITIS environment variables are set OR do a check to see if these applications are found with the `which` command.

Reply:
Added which command to check tools.